src/containers/Leave/index.tsx

```diff
@@ -48,7 +49,7 @@ const Leave: React.FC = () => {
   const { currentUserError, currentUserData, currentUserLoading } =
     useCurrentUser();
   const role = currentUserData?.data?.data?.getCurrentUser.role;
```

Comment:
can we clean this up in useCurrentUser, so that we donot have to go deep down to get role. Also why do we have getCurrentUser as a variable?